Library/Homebrew/requirements/language_module_requirement.rb

```diff
@@ -12,11 +12,26 @@ def initialize(language, module_name, import_name = nil)
 
   satisfy(:build_env => false) { quiet_system(*the_test) }
 
-  def message; <<-EOS.undent
-    Unsatisfied dependency: #{@module_name}
-    Homebrew does not provide #{@language.to_s.capitalize} dependencies; install with:
-      #{command_line} #{@module_name}
+  def message
+    s = ""
+
+    s += <<-EOS.undent
+      Unsatisfied dependency: #{@module_name}
+      Homebrew does not provide special #{@language.to_s.capitalize} dependencies; install with:
+        #{command_line} #{@module_name}
     EOS
+
+    if @language == :rbx
+      s
+    elsif $?.exitstatus == 127
+      s += <<-EOS.undent
+        You may need to:
+          brew install #{@language}
+      EOS
+    s
+    else
+    s
+    end
```

Comment:
In any case we should use `<<` for string concatenation instead of `+=`. The former is faster and more memory efficient because it doesn’t create a new string.